src/components/chat/w2wChat/chats/Chats.tsx

```diff
@@ -335,7 +335,7 @@ const SenderMessage = styled.div`
   box-sizing: border-box;
   position: relative;
   right: 34px;
-  max-width: 419px;
+  max-width: 650px;
```

Comment:
this has to be done for receiverMessage box too

Reply:
@mishramonalisha76 I added this change to `ReceivedMessage` as well.